.github/workflows/pinned_build.yaml

```diff
@@ -26,3 +26,45 @@ jobs:
           platform-file: .cicd/platforms.json
           password: ${{secrets.GITHUB_TOKEN}}
           package-name: builders
+
+  Build:
+    name: Build
+    needs: [d]
+    if: always() && needs.d.result == 'success'
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ubuntu18, ubuntu20, ubuntu22]
+    runs-on: ["self-hosted", "enf-x86-beefy"] # not sure if this warrants a different machine, but start here due to building
+    container: ${{ matrix.platform == 'ubuntu18' && 'ubuntu:bionic' || matrix.platform == 'ubuntu20' && 'ubuntu:focal' || 'ubuntu:jammy' }}
+    steps:
+      - name: Update and Install git
+        run: |
+          apt-get update
+          apt-get install -y git
+          git --version
+      - name: Clone leap
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Install dependencies
+        run: |
+          # https://github.com/actions/runner/issues/2033
+          chown -R $(id -u):$(id -g) $PWD
+          ./scripts/install_deps.sh
+      - name: Build Pinned Build
+        run: |
+          ./scripts/pinned_build.sh deps build "$(nproc)"
+          cd build
+          cpack
+          ls -l
+          cd ..
+      - name: Upload package
+        uses: actions/upload-artifact@v3
+        with:
+          name: leap-${{matrix.platform}}-amd64
+          path: build/leap*.deb
```

Comment:
`leap-dev.deb` is broken for pinned builds, so I'd suggest only uploading `leap.deb` as part of this artifact somehow.

I'm also unsure whether or not we should have the artifact name explicitly call out pinned (e.g. `leap-${{matrix.platform}}-pinned-amd64`).. but I think it's okay like this for now